bundles/org.eclipse.osgi.services/META-INF/MANIFEST.MF

```diff
@@ -2,56 +2,28 @@ Manifest-Version: 1.0
- org.osgi.service.event;version="1.4";uses:="org.osgi.framework",
- org.osgi.service.http;version="1.2.1";uses:="javax.servlet,javax.servlet.http",
- org.osgi.service.http.context;version="1.1";uses:="org.osgi.framework,javax.servlet.http",
- org.osgi.service.http.runtime;version="1.1";uses:="org.osgi.service.http.runtime.dto",
- org.osgi.service.http.runtime.dto;version="1.1";uses:="org.osgi.dto,org.osgi.framework.dto",
- org.osgi.service.http.whiteboard;version="1.1",
- org.osgi.service.log;version="1.5";uses:="org.osgi.framework",
- org.osgi.service.metatype;version="1.4";uses:="org.osgi.framework",
- org.osgi.service.provisioning;version="1.2",
- org.osgi.service.upnp;version="1.2",
- org.osgi.service.useradmin;version="1.1";uses:="org.osgi.framework",
- org.osgi.service.wireadmin;version="1.0.1";uses:="org.osgi.framework"
-Import-Package: javax.servlet;resolution:=optional,
- javax.servlet.http;resolution:=optional,
- org.osgi.dto;version="1.0",
- org.osgi.framework;version="1.6",
- org.osgi.framework.dto;version="1.8.0",
- org.osgi.service.cm;version="[1.6,1.7)",
- org.osgi.service.component;version="[1.4,1.5)",
- org.osgi.service.component.annotations;version="[1.3,1.4)",
- org.osgi.service.component.runtime; version="[1.4,1.5)",
- org.osgi.service.component.runtime.dto; version="[1.4,1.5)",
- org.osgi.service.device;version="[1.1,1.2)",
- org.osgi.service.event;version="[1.4,1.5)",
- org.osgi.service.http;version="[1.2,1.3)",
- org.osgi.service.http.context;version="[1.1,1.2)",
- org.osgi.service.http.runtime;version="[1.1,1.2)",
- org.osgi.service.http.runtime.dto;version="[1.1,1.2)",
- org.osgi.service.http.whiteboard;version="[1.1,1.2)",
- org.osgi.service.log;version="[1.5,1.6)",
- org.osgi.service.metatype;version="[1.4,1.5)",
- org.osgi.service.provisioning;version="[1.2,1.3)",
- org.osgi.service.upnp;version="[1.2,1.3)",
- org.osgi.service.useradmin;version="[1.1,1.2)",
- org.osgi.service.wireadmin;version="[1.0,1.1)",
- org.osgi.util.function;version="[1.0,2.0)",
- org.osgi.util.promise;version="[1.0,2.0)"
+Export-Package: org.osgi.service.component.annotations;version="1.3"
+Import-Package: org.osgi.annotation.versioning;version="[1.1.2,1.2.0)",
+ org.osgi.service.component.annotations;version="[1.3,1.4)"
 DynamicImport-Package: javax.servlet,
  javax.servlet.http
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Automatic-Module-Name: org.eclipse.osgi.services
+Require-Bundle: org.osgi.service.cm;bundle-version="[1.6.0,1.7.0)";visibility:=reexport,
+ org.osgi.service.component;bundle-version="[1.5.0,1.6.0)";visibility:=reexport,
+ org.osgi.service.device;bundle-version="[1.1.0,1.2.0)";visibility:=reexport,
+ org.osgi.service.event;bundle-version="[1.4.0,1.5.0)";visibility:=reexport,
+ org.osgi.service.http;bundle-version="[1.2.0,1.3.0)";visibility:=reexport,
+ org.osgi.service.http.whiteboard;bundle-version="[1.1.0,1.2.0)";visibility:=reexport,
+ org.osgi.service.log;bundle-version="[1.5.0,1.6.0)";visibility:=reexport,
```

Comment:
As I noted in https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/235 we are going to need to avoid pulling in this log bundle.

Reply:
From https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/235#discussion_r858669741:
> We really want to avoid installing the OSGi WG `org.osgi.service.log` bundle because I suspect it only exports without importing the log packages.

```
Bundle-SymbolicName: org.osgi.service.log
Bundle-Version: 1.5.0.202007221806
Export-Package: org.osgi.service.log;version="1.5";uses:="org.osgi.framework",org.osgi.service.log.admin;version="1.0";uses:="org.osgi.service.log"
Import-Package: org.osgi.framework;version="[1.8,2)",org.osgi.service.log;version="[1.5,2)"
```
The Manifest shows that it is also importing `org.osgi.service.log`, nevertheless I think we should avoid a second unnecessary provider of that package.

> 
> What happens today with the `org.eclipse.osgi.services` bundle is that it ends up always importing the `org.osgi.service.log` package from the framework and never gets selected as an exporter of the package.
> 
> This is another package in `org.eclipse.osgi.services` (like `component.annotations`) that we may just want to keep as-is until we can deprecate and remove the bundle altogether.

That sounds like a good way to go.
I'm planning to remove all require-Bundles of the Plug-ins that used to contain OSGi-sources in the Eclipse-SDK anyway subsequently.

Reply:
The removal of `org.osgi.service.log` is reverted.